src/constants.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+const FUNC_INSTALL				= 0x0000800;
+const FUNC_BASIC				= 0x0001000;
+const FUNC_STACK_TRACE			= 0x0002000; // affects stack traces
+const FUNC_REMOTE				= 0x0004000; // affects remote debugging
+const FUNC_FUNCTION_TRACE		= 0x0008000; // affects function traces
+const FUNC_VAR_DUMP				= 0x0010000; // affects overloaded var_dump function
+const FUNC_PROFILER				= 0x0020000; // affects overloaded var_dump function
+const FUNC_CODE_COVERAGE	  	= 0x0040000;
```

Comment:
I should apologize. When I said before that I prefer tabs, I meant that *for indentation*, but not for alignment. I'll make a PR against this PR to fix that.

Reply:
I've now done that btw.